project/Dependencies.scala

```diff
@@ -30,9 +30,6 @@ object Dependencies {
     lazy val partyProxyClient =
       namespace %% "interop-be-party-registry-proxy-client" % partyProxyVersion
 
-    lazy val userRegistryManagementClient =
-      namespace %% "pdnd-interop-uservice-user-registry-management-client" % userRegistryManagementVersion
```

Comment:
you can also delete `userRegistryManagementVersion` val in `Versions.scala` file